checker/docker-compose.yml.ipv4

```diff
@@ -0,0 +1,12 @@
+  
+version: '3.6'
+services:
+  casino-backend:
+    build: .
+    volumes:
+      - .:/checker
```

Comment:
is this volume required? The original backend doesn't have it (?)

Reply:
I don't think it's required, thank you. It's removed in https://github.com/enowars/enowars3-service-casino/pull/1/commits/130ead88888bcf727e997cabe91e26e690895685.